integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRemoteConsole.java

```diff
@@ -0,0 +1,219 @@
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@DisplayName("Test WebLogic remote console connecting to mii domain")
+@IntegrationTest
+class ItRemoteConsole {
+
+  private static String domainNamespace = null;
+
+  // domain constants
+  private static final String domainUid = "domain1";
+  private static final String clusterName = "cluster-1";
+  private static final int replicaCount = 1;
+  private static final String adminServerPodName = domainUid + "-" + ADMIN_SERVER_NAME_BASE;
+  private static final String managedServerPrefix = domainUid + "-" + MANAGED_SERVER_NAME_BASE;
+  private static LoggingFacade logger = null;
+
+  /**
+   * Get namespaces for operator and WebLogic domain.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    final String opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // create a basic model in image domain
+    createAndVerifyMiiDomain();
+
+  }
+
+  /**
+   * Verify WLS Remote Console installation is successful.
+   * Verify k8s WebLogic domain is accessible through remote console.
+   */
+  @Test
+  @DisplayName("Verify WLS Remote Console installation is successful")
+  public void testWlsRemoteConsoleInstallation() {
+
+    assertTrue(installAndVerifyWlsRemoteConsole(), "Remote Console installation failed");
+
+    int nodePort = getServiceNodePort(
+        domainNamespace, getExternalServicePodName(adminServerPodName), "default");
+    assertTrue(nodePort != -1,
+        "Could not get the default external service node port");
+    logger.info("Found the default service nodePort {0}", nodePort);
+    logger.info("The K8S_NODEPORT_HOST is {0}", K8S_NODEPORT_HOST);
+    String curlCmd1 = "curl -s -L -v --user weblogic:welcome1 --show-error --noproxy '*' "
+        + " http://" + K8S_NODEPORT_HOST + ":" + nodePort
+        + "/management/weblogic --write-out %{http_code} -o /dev/null";
+    logger.info("Executing default nodeport curl command {0}", curlCmd1);
+    assertTrue(callWebAppAndWaitTillReady(curlCmd1, 10), "Calling web app failed");
+    logger.info("WebLogic domain is accessible through remote console");
+
+  }
+
+  /**
+   * Shutdown WLS Remote Console.
+   */
+  @AfterAll
+  public void tearDownAll() {
```

Comment:
Check if SKIP_CLEANUP variable is set, if set skip the console shutdown

Reply:
CLEANUP can't shutdown console

Reply:
Add an extra method in clean up framework to shutdown console if started by the test.

Reply:
Change the logic as follows ...

if (System.getenv("SKIP_CLEANUP") == null
        || (System.getenv("SKIP_CLEANUP") != null
        && System.getenv("SKIP_CLEANUP").equalsIgnoreCase("false"))) {
           assertTrue(shutdownWlsRemoteConsole(), "Remote Console shutdown failed");
    }

Reply:
Done

Reply:
I do not see the change on the latest commit